README.md

```diff
@@ -1549,6 +1549,37 @@ Query `posts(limit:10,page:1){data{id},total,per_page}` might return
 Note that you need to add in the extra 'data' object when you request paginated resources as the returned data gives you
 the paginated resources in a data object at the same level as the returned pagination metadata.
 
+Simple Pagination will be used, if a query or mutation returns a `SimplePaginationType`.
```

Comment:
I think it would be nice to link to the Laravel docs here
```suggestion
[Simple Pagination](https://laravel.com/docs/pagination#simple-pagination) will be used, if a query or mutation returns a `SimplePaginationType`.
```